app/bundles/CoreBundle/Helper/DateRelativeParser.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Helper;
+
+class DateRelativeParser
+{
+    /** @var array */
```

Comment:
Everywhere in Mautic this comment is on 3 lines. It would be really nice to have it the same everywhere.